src/util/std_expr.cpp

```diff
@@ -223,3 +224,34 @@ const exprt &object_descriptor_exprt::root_object() const
 
   return *p;
 }
+
+void member_exprt::validate(
+  const exprt &expr,
+  const namespacet &ns,
+  const validation_modet vm)
+{
+  check(expr, vm);
+
+  const auto &member_expr = to_member_expr(expr);
```

Comment:
:question: why not take a `member_exprt` as argument instead, and let the caller call `to_member_expr` if needed?

Reply:
Appears to be normal for `validate` methods, again change en masse or not at all